images/php/Alpine/supervisord.conf

```diff
@@ -0,0 +1,20 @@
+[supervisord]
+nodaemon=true
+
+[program:apache2]
+command=/usr/sbin/httpd -D FOREGROUND
+autorestart=true
+stdout_logfile=/var/log/apache2/access.log
```

Comment:
log file needs to change to log to stdout (maybe /proc/self/fd/2 or something

https://stackoverflow.com/questions/45645758/supervisord-disable-log-files-or-use-logfile-dev-stdout